compio-runtime/src/runtime/op.rs

```diff
@@ -42,3 +42,37 @@ impl<T: OpCode> Drop for OpFuture<T> {
         }
     }
 }
+
+#[derive(Debug)]
+pub struct OpFlagsFuture<T: OpCode> {
```

Comment:
This struct makes the `OpFuture` type needless.

Reply:
removed